pkg/webhook/inject/inject_test.go

```diff
@@ -27,14 +27,16 @@ import (
 	"github.com/chaos-mesh/chaos-mesh/pkg/webhook/config"
 )
 
+var i Injector
```

Comment:
do not introduce a new package level variable in the test. instant new one in the test codes.

Reply:
You could initialize a new Injector between `Describe` and `Context` statements.